Index: src/main/java/org/shalai/ivan/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.shalai.ivan;\n\n// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,\n// then press Enter. You can now see whitespace characters in your code.\npublic class Main {\n    public static void main(String[] args) {\n        // Press Opt+Enter with your caret at the highlighted text to see how\n        // IntelliJ IDEA suggests fixing it.\n        System.out.printf(\"Hello and welcome!\");\n\n        // Press Ctrl+R or click the green arrow button in the gutter to run the code.\n        for (int i = 1; i <= 10; i++) {\n\n            // Press Ctrl+D to start debugging your code. We have set one breakpoint\n            // for you, but you can always add more by pressing Cmd+F8.\n            System.out.println(\"i = \" + i);\n        }\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/shalai/ivan/Main.java b/src/main/java/org/shalai/ivan/Main.java
--- a/src/main/java/org/shalai/ivan/Main.java	
+++ b/src/main/java/org/shalai/ivan/Main.java	
@@ -1,19 +1,7 @@
 package org.shalai.ivan;
 
-// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
-// then press Enter. You can now see whitespace characters in your code.
 public class Main {
     public static void main(String[] args) {
-        // Press Opt+Enter with your caret at the highlighted text to see how
-        // IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        // Press Ctrl+R or click the green arrow button in the gutter to run the code.
-        for (int i = 1; i <= 10; i++) {
-
-            // Press Ctrl+D to start debugging your code. We have set one breakpoint
-            // for you, but you can always add more by pressing Cmd+F8.
-            System.out.println("i = " + i);
-        }
     }
+    
 }
\ No newline at end of file
